store/extensions/G-itemViewer/extension.json

```
@@ -0,0 +1,35 @@
+{
+  "title": "G-itemViewer",
+  "description": "A Habbo-styled GUI used to count and inspect Inventory and Room furni and wallitems",
+  "authors": [
+    {
+      "name": "madlad",

```

Remove the comma at the end of this line
```suggestion
      "name": "madlad"
```